website/docs/infrastructure/archive-data.md

```diff
@@ -0,0 +1,160 @@
+# Archiving Data from Decommissioned Projects
+
+## Overview
+
+When a project is decommissioned, it's crucial to maintain data for legal and auditing purposes while minimizing storage costs. This guide provides a comprehensive approach to data archiving with a focus on Azure storage solutions.
+
+## Best Practices for Data Archiving
+
+### Recommended Storage Account Configuration
+
+The optimal storage solution balances flexibility, support, and cost-effectiveness with the following characteristics:
+
+1. **Blob Storage Configuration**
+   - Use Archive storage tier
+   - Disable internet access
+   - Enforce authentication via Entra ID (disable access keys)
+   - Implement legal hold policies
+   - Configure multi-AZ replication in the primary region
+   - Set up single-AZ replication in a secondary region
+   - Enable object replication between regions
+   - Apply read-only lock
+
+### Sample Terraform Configuration
+
+```hcl
+resource "azurerm_storage_account" "backup_primary" {
```

Comment:
what about adding an `azure_storage_archive` module that _eventually_ extends [azure_storage_account](https://github.com/pagopa/dx/tree/main/infra/modules/azure_storage_account) ? or a special tier ('archive') in the existings `azure_storage_account` module?

Reply:
We should add support of entities as well, but we can think about it